### DISCUSSION TITLE
Hello Linux!!

### DISCUSSION THREAD

**Discussion by abdelrahman1215** [abdelrahman1215/simple-txt#2] [Announcements]
* **made the program availble for linux**
* fixed some bugs
* added a ln command that move the cursor to the nth line
* added a col command that moves the cursor to the nth column

<hr /><em>This discussion was created from the release <a href='https://github.com/abdelrahman1215/simple-txt/releases/tag/v0.3-Alpha'>Hello Linux!!</a>.</em>

**Comment by roxy1234** [comment]
Hello, I'm a newbie. Just enrolled a course at [Linux](https://yellowtail.tech/courses/linux/linux-training/). I would like to ask if I made a right decision or I need to enroll to another course or program. Thanks!
